px-threshold-bar.html

```diff
@@ -230,27 +230,32 @@
       }
     },
 
+    /**
+     * Calculates in pixels the padding that should be applied to all left-aligned elements
+     */
+    _calculatePaddingWidth: function() {
+      if (Polymer.dom(this.root).querySelector('.scale__value--first')) {
```

Comment:
Minor change at your discretion: wouldn't hurt to throw `Polymer.dom(this.root).querySelector('.scale__value--first')` into a variable. Would make it a lot easier to read.

Reply:
Good point, updated.